cost-analyzer/templates/aggregator-statefulset.yaml

```diff
@@ -1,5 +1,5 @@
-{{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostAggregator) }}
-{{- if .Values.kubecostAggregator.enabled }}
+{{- if and (not .Values.agent) (not .Values.cloudAgent) }}
```

Comment:
can we also template out the selector labels here to standardize them?

Reply:
I believe they are templated? I'm not sure where you're pointing.

Reply:
Ah, indeed! Great find, who knows why this is the only place not using the template that already exists. Fixed.